code/style.css

```diff
@@ -1,58 +1,126 @@
+@import url('https://fonts.googleapis.com/css2?family=Grenze+Gotisch:wght@300;500&display=swap');
+
 body {
-  margin: 0;
   font-family: "Courier New", Courier, monospace;
+  background-image: url(./Images/background-unsplash.jpg);
+  background-size: cover;
+}
+
+
+h1 {
+  margin-top: 100px;
+  font-family: 'Grenze Gotisch', cursive;
+  font-size: 60px;
+  color: rgb(255, 255, 255);
+  text-shadow: 5px 5px 5px rgba(128, 128, 128, 0.753);
+  text-align: center;
 }
 
-p {
-  margin: 1px;
+
+#mystery {
```

Comment:
Here you have mixed styling classes and id's, maybe it would be better to just keep to styling one or the other, you can always add a class to a html-element for the styling purpose, and let the id's be exclusive for the javascript.

Reply:
thanks, good point! =)